tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -32,10 +32,13 @@ import (
 var testBucket = flag.String("testbucket", "", "The GCS bucket used for the test.")
 
 var (
-	testDir string
-	binFile string
-	logFile string
-	mntDir  string
+	testDir  string
+	binFile  string
+	logFile  string
+	mntDir   string
+	tmpDir   string
+	fileName string
```

Comment:
Please incorporate the comments from other PR in this.

Reply:
Done